### ISSUE TITLE
[dev.icinga.com #12620] Allow to configure a certificate for Icinga 2 API command transport

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#2541]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/12620

**Created by tobiasvdk on 2016-09-01 18:40:45 +00:00**

Assignee: _(none)_
Status: _New_ 
Target Version: _Backlog_
Last Update: _2016-10-04 12:55:01 +00:00 (in Redmine)_

---

Currently it's only possible to configure username and password. Please also allow to use a certificate (file) instead of a password.

**Comment by Al2Klimov** [comment]
blocked by #3016

**Comment by Al2Klimov** [comment]
closing in favor of #3036
